gensim/test/test_word2vec.py

```diff
@@ -461,6 +462,141 @@ def test_cbow_neg(self):
                                   min_count=5, iter=10, workers=2, sample=0)
         self.model_sanity(model)
 
+    def test_most_similar_among_CBOW(self):
```

Comment:
Sure, Removed extra tests; using CBOW method to check functionality.

That second part might be important given @tmylk's comment
> KeyedVectors is the way to go.

What do you think @tmylk ?